Sources/Rownd/Views/HubWebView/HubWebViewController.swift

```diff
@@ -317,18 +317,17 @@ extension HubWebViewController: WKScriptMessageHandler, WKNavigationDelegate {
                 )
                 
             case .triggerSignInWithGoogle:
-                //self.hubViewController?.height()
                 var signInWithGoogleMessage: MessagePayload.TriggerSignInWithGoogleMessage? = nil
                 if case .triggerSignInWithGoogle(let message) = hubMessage.payload {
                     signInWithGoogleMessage = message
                 }
-//                self.hubViewController?.hide()
-                Rownd.requestSignIn(
-                    with: .googleId,
-                    signInOptions: RowndSignInOptions(
-                        intent: signInWithGoogleMessage?.intent
+                Task {
+                    await Rownd.googleSignInCoordinator.signIn(
```

Comment:
Could we not sill call `Rownd.requestSignIn()`, or does that not make sense anymore? I'm thinking something like:

```swift
Rownd.requestSignIn(
    with: .googleId,
    signInOptions: RwondSignInOptions(
        intent: signInWithGoogleMessage?.intent,
        hint: signInWithGoogleMessage?.hint
    )
)

```

Reply:
Yeah we can do that. I was thinking of leaving `hint` private. Would we want to make it public?

Reply:
Sounds good. Updated!